distributions/distribution-resources/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -28,7 +28,7 @@ log4j.appender.out.maxBackupIndex=10
 # File appender - events.log
 log4j.appender.event=org.apache.log4j.RollingFileAppender
 log4j.appender.event.layout=org.apache.log4j.PatternLayout
-log4j.appender.event.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss.SSS} [%-20.20c{1}] - %m%n
+log4j.appender.event.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss.SSS} [%c{1}] - %m%n
```

Comment:
Yeah, this is what I don't like - the log messages are not aligned - so I would indeed prefer to change it back (or probably to 26, so that GroupItemStateChangedEvent fits in; this actually did not exist at the time I set it to 20).